diff --git a/src/borg/archiver.py b/src/borg/archiver.py
index 8b898706..d4bf9cd0 100644
--- a/src/borg/archiver.py
+++ b/src/borg/archiver.py
@@ -586,7 +586,7 @@ def create_inner(archive, cache, fso):
                         self.print_warning_instance(BackupWarning(path, e))
                         status = 'E'
                     if status == 'C':
-                        self.print_warning_instance(FileChangedWarning(path))
+                        print(f'{path}: file changed while we backed it up', file=sys.stderr)
                     self.print_file_status(status, path)
                 if args.paths_from_command:
                     rc = proc.wait()
@@ -840,7 +840,7 @@ def _rec_walk(self, *, path, parent_fd, name, fso, cache, matcher,
             self.print_warning_instance(BackupWarning(path, e))
             status = 'E'
         if status == 'C':
-            self.print_warning_instance(FileChangedWarning(path))
+            print(f'{path}: file changed while we backed it up', file=sys.stderr)
         if not recurse_excluded_dir:
             self.print_file_status(status, path)
 
